README.md

```diff
@@ -1,86 +1,56 @@
-[Cozy](https://cozy.io) Drive for Desktop <sup>(beta)</sup>
-===========================================================
+# [Cozy](https://cozy.io) Drive for Desktop <sup>(beta)</sup>
+
+[![Cozy Logo](https://raw.github.com/cozy/cozy-setup/gh-pages/assets/images/happycloud.png)](https://cozy.io)
 
 [![Latest release](https://img.shields.io/github/release/cozy-labs/cozy-desktop/all.svg)](https://github.com/cozy-labs/cozy-desktop/releases)
 [![Licence](https://img.shields.io/github/license/cozy/cozy-drive.svg)](https://github.com/cozy/cozy-drive/blob/master/LICENSE)
 <br>
 [![Windows build](https://img.shields.io/appveyor/ci/cozy/cozy-desktop/master.svg?label=windows)](https://ci.appveyor.com/project/cozy/cozy-desktop/branch/master)
-[![Linux & macOS build]( 	https://img.shields.io/travis/cozy-labs/cozy-desktop/master.svg?label=mac/linux)](https://travis-ci.org/cozy-labs/cozy-desktop/branches)
+[![Linux & macOS build](https://img.shields.io/travis/cozy-labs/cozy-desktop/master.svg?label=mac/linux)](https://travis-ci.org/cozy-labs/cozy-desktop/branches)
 [![Code climate](https://img.shields.io/codeclimate/github/cozy-labs/cozy-desktop.svg)](https://codeclimate.com/github/cozy-labs/cozy-desktop)
 [![Coverage](https://img.shields.io/codecov/c/github/cozy-labs/cozy-desktop/master.svg?label=coverage)](https://codecov.io/gh/cozy-labs/cozy-desktop/list/master)
```

Comment:
visually these two badges display the same label
![image](https://user-images.githubusercontent.com/701648/44844862-40d51b00-ac4c-11e8-82fb-074f19eaa792.png)

what do we do?

Reply:
Possible solutions I can think of:

1. Drop the Code Climate badge
2. Find a way to display the Code Climate rating
3. Use Code Climate for coverage and drop Codecov

I don't have a strong preference.

Reply:
@sebn 
I found another badge for code climate rating, are we ok to replace the actual codeclimate one with this one

![](https://img.shields.io/codeclimate/maintainability-percentage/cozy-labs/cozy-desktop.svg)